src/accelerate/big_modeling.py

```diff
@@ -387,8 +387,10 @@ def dispatch_model(
         retie_parameters(model, tied_params)
     else:
         device = list(device_map.values())[0]
-        if device != "disk":
+        if device != "disk" and not getattr(model, "is_quantized", False):
```

Comment:
we should maybe check with the attributes `is_loaded_in_8bit` or `is_loaded_in_4bit` as `is_quantized` has been only recently introduced. WDYT?